velox/dwio/common/ScanSpec.h

```diff
@@ -255,6 +255,12 @@ class ScanSpec {
   // This may change as a result of runtime adaptation.
   bool hasFilter() const;
 
+  /// Assume this field is read as null constant vector (usually due to missing
+  /// field), check if any filter in the struct subtree would make the whole
+  /// vector to be filtered out.  Return false when the whole vector should be
+  /// filtered out.
+  bool testNull() const;
```

Comment:
Maybe I'm not used to the convention. But it appears to be more intuitive to me to call it something like filterNullConstant() and return true when the whole vector should be filtered out.

Reply:
The test prefix method is the contention used in `Filter` classes